src/cascade/executor/epiviz_runner.py

```diff
@@ -173,6 +204,21 @@ def has_random_effects(model):
 
 
 def main(args):
+    """
+     1. Parse arguments to the command line. The GUI calls a shell script
+        which passes the model version ID to the EpiViz Runner.
+
+     2. Construct an Execution Context, which tells EpiViz Runner where it can find
+        directories, files, and databases on the cluster.
+
+     3. Load Settings, which are the values you entered into EpiViz-AT on the web page.
+
+     4. Using the Settings and the Execution Context, fill out the data and
+        any parameters for Dismod-AT, as described in
+        :ref:`build-model-from-epiviz-settings`.
+
+     5. Put that data into a file and run Dismod-AT on that file.
+    """
     ec = make_execution_context()
```

Comment:
Just saying, there is a "1. Parse ..." above, and just after the comment, we start with "2. Construct an EC"
I see in the next step, we use args.